packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -188,6 +188,7 @@ export interface LocalSecondaryIndexProps extends SecondaryIndexProps {
 export class Table extends Resource {
   /**
    * Permits an IAM Principal to list all DynamoDB Streams.
+   * @deprecated Use grantTableListStream() for more granular permission
```

Comment:
Move this below `@param`, with an empty line in between:

```ts
  /**
   * Permits an IAM Principal to list all DynamoDB Streams.
   *
   * @param grantee The principal (no-op if undefined)
   *
   * @deprecated Use grantTableListStream() for more granular permission
   */
```

Reply:
Also, you can use the `{@link grantTableListStream}` notation, and the name of the new method is `grantTableListStream`**s**.